lab_1_classify_by_unigrams/main.py

```diff
@@ -85,6 +132,35 @@ def compare_profiles(
     'freq' in arguments, None is returned
     """
 
+    if (not isinstance(unknown_profile, dict)
+            or not isinstance(profile_to_compare, dict)):
+        return None
+    if ("name" not in unknown_profile
+            or "freq" not in unknown_profile
+            or "name" not in profile_to_compare
+            or "freq" not in profile_to_compare):
+        return None
+
+    unknown_freq, compare_freq = {}, {}
+    if isinstance(unknown_profile, dict):
+        unknown_freq = unknown_profile["freq"]
+    if isinstance(profile_to_compare, dict):
+        compare_freq = profile_to_compare["freq"]
+    unknown_keys, compare_keys = list(unknown_freq.keys()), list(compare_freq.keys())
```

Comment:
avoid `a, b = 5` cases, write:
`a = 5`
`b = 5`

Reply:
got it